app/models/user.rb

```diff
@@ -177,6 +177,10 @@ def self.authenticate_from_form(params, specific_user_login = false)
       unless user.has_this_password?(params[:password])
         user.errors.add(:base, auth_fail_message)
       end
+
+      if user.closed?
+        user.errors.add(:base, _('This account has been closed.'))
```

Comment:
Not sure this message should be shown without the correct password being entered for the user account. Could combine with the conditional above to prevent this.

Reply:
+1, Fixed in f8cbe27